packages/domain/lib/src/di/domain_dependencies.dart

```diff
@@ -10,6 +10,11 @@ final inject = Needle.instance;
 void initDomainDependencies() {
   initMappers();
   initUseCases();
+  initPlugins();
+}
+
+void initPlugins() {
+  inject.registerFactory<ShareMovie>(() => ShareMovie());
```

Comment:
maybe create singleton?

Reply:
no need anymore)) sergey said to create a separate plugin and publish it at pub.dev =)